config/field_test.yml

```
@@ -28,10 +28,38 @@
 ################################################################################
 experiments:
   # NOTE: Our feed strategy testing experiment must begin with "feed_strategy"
+  feed_strategy_starting_20230814:
+    # NOTE: Required as we want only want to consider for conversion events that
+    #       occurred on or after the given start_date.
+    started_at: 2023-08-15
+    variants:
+      - 20230511-variant-b
+      - 20230814-variant-a
+      - 20230814-variant-b
+      - 20230814-variant-c
+    weights:
+      - 55
+      - 15
+      - 15
+      - 15
+    goals:
+      - user_creates_pageview
+      - user_creates_article_reaction
+      - user_creates_comment
+      - user_views_pages_on_at_least_two_different_days_within_a_week
+      - user_views_pages_on_at_least_four_different_days_within_a_week
+      - user_creates_article_reaction_on_four_different_days_within_a_week
+      - user_creates_comment_on_at_least_four_different_days_within_a_week
+      - user_views_pages_on_at_least_three_different_hours_within_a_day
+      - user_views_pages_on_at_least_four_different_hours_within_a_day
+      - user_views_pages_on_at_least_twelve_different_hours_within_five_days
+      - user_views_pages_on_at_least_nine_different_days_within_two_weeks
   feed_strategy_starting_20230511:
     # NOTE: Required as we want only want to consider for conversion events that
     #       occurred on or after the given start_date.
     started_at: 2023-05-11
+    ended_at: 2023-08-14
```

Should the new start date (on line 34) and the end date of the old experiment be the same? Not sure how important the accuracy is.

```suggestion
    ended_at: 2023-08-15
```